content/en/data_jobs/databricks.md

```diff
@@ -74,12 +74,12 @@ Optionally, you can also set other init script parameters and Datadog environmen
 
 | Variable                 | Description                                                                                                                                                      | Default |
 |--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
-| DD_API_KEY               | Your [Datadog API key][1]                                                                                                                                        |         |
-| DD_SITE                  | Your [Datadog site][2]                                                                                                                                           |         |
-| DATABRICKS_WORKSPACE     | Name of your Databricks Workspace. It should match the name provided in the [Datadog-Databricks integration step](#configure-the-datadog-databricks-integration) |         |
-| DRIVER_LOGS_ENABLED      | To collect spark driver logs in Datadog                                                                                                                          | false   |
-| WORKER_LOGS_ENABLED      | To collect spark workers logs in Datadog                                                                                                                         | false   |
-
+| DD_API_KEY               | Your [Datadog API key][1].                                                                                                                                        |         |
+| DD_SITE                  | Your [Datadog site][2].                                                                                                                                           |         |
+| DATABRICKS_WORKSPACE     | Name of your Databricks Workspace. It should match the name provided in the [Datadog-Databricks integration step](#configure-the-datadog-databricks-integration). Make sure to enclose the name in double quotes if it contains whitespace. |         |
```

Comment:
@yiliangzhou should we not just put double quotes around the value in the init script/spark params templates so customers see it there too if they don't read this? Any reason not to do that?

Reply:
agree. will update those as well.